build_tools/release_automation.py

```diff
@@ -240,7 +240,7 @@ def update_sasmodels_init(version):
     """
     init_file = os.path.join('sasmodels', 'sasmodels', '__init__.py')
     output_lines = []
-    with open(init_file, 'r') as f:
+    with open(init_file) as f:
```

Comment:
Personally, I'm not a fan of removing the explicit setting of read in the calls to open, but understand it's the default, so understand why this is.